src/DIRAC/ConfigurationSystem/Client/Helpers/Operations.py

```diff
@@ -67,7 +67,38 @@
          Operations(group=thisIsAGroupOfVO_X).getValue('someSectionName/someOptionX')
 
     3. if no VO nor group is provided, the VO will be guessed from the proxy,
-    but this works iff the object is instantiated by a proxy (and not, e.g., using a server certificate)
+    but this works if the object is instantiated by a proxy (and not, e.g., using a server certificate)
+
+    You can also specify the main section::
+
+      Operations(vo=VOName, mainSection='/mainSectionName').getValue('someSection/someOption')
+
+    in this case, the configuration will be searched in the following sections with the name 'mainSectionName'::
```

Comment:
I am not sure I understand. If there are many sections named "mainSectionName", and in each of them there's a "someSection/someOption", what's returned?

Reply:
I updated the documentation to more details.

The merger takes place in the following order (which is almost no different from the Operations usual):
- /`mainSectionName`/...
- /Operations/`setup`/`mainSectionName`/...
- /Operations/`vo`/Defaults/`mainSectionName`/...
- /Operations/`vo`/`setup`/`mainSectionName`/...